specification/workloads/Workloads.SAPVirtualInstance.Management/SAPApplicationServerInstance.tsp

```diff
@@ -60,7 +60,11 @@ interface SAPApplicationServerInstances {
    */
   start is ArmResourceActionAsync<
     SAPApplicationServerInstance,
-    StartRequest,
+    {
+      @doc("SAP Application server instance start request body.")
+      @body
+      body?: StartRequest;
```

Comment:
Could you help me understand why this POST body could be optional?

Reply:
As far as I know, no guideline about this.

There could be valid POST that perform an action, even without a request body; but given request body, the action could be performed a bit differently.

Reply:
Hello @MaryGao @weidongxu-microsoft , yes, in service, we have some default execution of StartRequest when body is not passed. And execution changes when options for StartRequest is passed in the body.

Reply:
Got it and thanks all!